### PR TITLE
Use more specific logs related rules

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Use more specific rules regarding log files

### Motivation

The logs_removed and logs_altered are amongst the noisiest rules

### CODE REVIEW

**Review by Gui774ume — APPROVED** [review]
lgtm

File: runtime/default.policy [hunk 1]
```diff
@@ -62,132 +94,153 @@ rules:
...
-      (chmod.filename =~ "/etc/*" ||
-      chmod.filename =~ "/sbin/*" || chmod.filename =~ "/usr/sbin/*" ||
-      chmod.filename =~ "/usr/local/sbin/*" || chmod.filename =~ "/usr/local/bin/*" ||
-      chmod.filename =~ "/var/log/*" || chmod.filename =~ "/usr/lib/*") &&
-      process.filename not in CONTAINER_PROCESSES
+      (chmod.file.path =~ "/etc/*" ||
+      chmod.file.path =~ "/sbin/*" || chmod.file.path =~ "/usr/sbin/*" ||
+      chmod.file.path =~ "/usr/local/sbin/*" || chmod.file.path =~ "/usr/local/bin/*" ||
+      chmod.file.path =~ "/var/log/*" || chmod.file.path =~ "/usr/lib/*") &&
+      process.file.path not in CONTAINER_PROCESSES
```
Comment (n06) [inline runtime/default.policy:184]: Should we allow-list package managers here? We do it for the creation of new binaries, so this feels like it follows a similar pattern.

**Review by n06 — APPROVED** [review]
Looks good to me, with one small suggestion for further tuning of permissions_changed

**Comment by n06** [comment]
Another comment: I think we need to remove the macros, and in-line the data instead. We don't have UI support for macros yet

File: runtime/default.policy (hunk 1 above)
Comment (lebauce) [reply runtime/default.policy]: Agreed

**Comment by lebauce** [comment]
@n06 If support for macros in the backend is coming soon, we could probably keep in the agent, it would make the rules a lot easier to understand.

**Pull request merged**